### PR TITLE
docs: sqlite migration example

### PR DESCRIPTION
**Pull request by jlarmstrongiv**
Migration example using SQLite data types.  Parts of the PostgreSQL example are not valid SQLite syntax, such as `serial` and `now()`.

I also used the plain SQLite [data types](https://www.sqlite.org/datatype3.html), instead of their type affinity.  If you would like to use affinity types, use this snippet instead:

```ts
import { Kysely, sql } from "kysely";

export async function up(db: Kysely<any>): Promise<void> {
  await db.schema
    .createTable("person")
    .addColumn("id", "integer", (col) => col.primaryKey())
    .addColumn("first_name", "varchar", (col) => col.notNull())
    .addColumn("last_name", "varchar")
    .addColumn("gender", "varchar(50)", (col) => col.notNull())
    .addColumn("created_at", "timestamp", (col) =>
      col.defaultTo(sql`CURRENT_TIMESTAMP`).notNull()
    )
    .execute();

  await db.schema
    .createTable("pet")
    .addColumn("id", "integer", (col) => col.primaryKey())
    .addColumn("name", "varchar", (col) => col.notNull().unique())
    .addColumn("owner_id", "integer", (col) =>
      col.references("person.id").onDelete("cascade").notNull()
    )
    .addColumn("species", "varchar", (col) => col.notNull())
    .execute();

  await db.schema
    .createIndex("pet_owner_id_index")
    .on("pet")
    .column("owner_id")
    .execute();
}

export async function down(db: Kysely<any>): Promise<void> {
  await db.schema.dropTable("pet").execute();
  await db.schema.dropTable("person").execute();
}
```

### CODE REVIEW

File: site/docs/migrations.mdx [hunk 1]
```diff
@@ -72,6 +72,46 @@ export async function down(db: Kysely<any>): Promise<void> {
...
 
+## SQLite migration example
+
+```ts
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('person')
+    .addColumn('id', 'integer', (col) => col.primaryKey())
```
Comment (igalklebanov) [inline site/docs/migrations.mdx:83]: missing auto increment..

```suggestion
    .addColumn('id', 'integer', (col) => col.primaryKey().autoIncrement())
```

File: site/docs/migrations.mdx [hunk 2]
```diff
@@ -72,6 +72,46 @@ export async function down(db: Kysely<any>): Promise<void> {
...
+    .addColumn('last_name', 'text')
+    .addColumn('gender', 'text', (col) => col.notNull())
+    .addColumn('created_at', 'text', (col) =>
+      col.defaultTo(sql`CURRENT_TIMESTAMP`).notNull()
+    )
+    .execute()
+
+  await db.schema
+    .createTable('pet')
+    .addColumn('id', 'integer', (col) => col.primaryKey())
```
Comment (igalklebanov) [inline site/docs/migrations.mdx:94]: missing auto increment..

```suggestion
    .addColumn('id', 'integer', (col) => col.primaryKey().autoIncrement())
```

**Review by igalklebanov — COMMENTED** [review]
Looks good! 💪 

Thank you! 🙏 

Left some suggestions..

**Comment by jlarmstrongiv** [comment]
@igalklebanov I don’t usually use autoincrement with SQLite—according to the docs:

> The AUTOINCREMENT keyword imposes extra CPU, memory, disk space, and disk I/O overhead and should be avoided if not strictly needed. It is usually not needed.
> https://www.sqlite.org/autoinc.html

I find that I don’t need to “prevent the reuse of ROWIDs from previously deleted rows”, so the default `INTEGER PRIMARY KEY` is my go-to choice for ROWID generation.

If you would still like `.autoIncrement()`, I can add it to the PR.

**Review by igalklebanov — APPROVED** [review]
TIL 🧐 

Approved! [🟢 ⚫ ⚫ ]

**Pull request merged**